a_locs_poly_setup/src/get_polygons.R

```diff
@@ -22,7 +22,7 @@ get_polygons <-  function(HUC, minimum_sqkm) {
   
   # check to see if huc4 exists yet
   file_list <- list.files("a_locs_poly_setup/out/", pattern = c("^huc4.*\\.gpkg$"))
-  if (length(file_list) == 0 | any(!grepl(huc4, file_list))) {
+  if (length(file_list) == 0 | all(!grepl(huc4, file_list))) {
```

Comment:
can the `huc4` object be a vector, or is is always just a single value? Does this condition produce an error if it is a vector?

Reply:
Turns out this usually IS a vector, and it's why it was causing issues before when I used `any()`, where even if the right file was in there, it would re run the download, which I didn't want. This was a logic correction on my part!